shared/macros-oval.jinja

```diff
@@ -230,3 +247,19 @@
         definition_ref="package_openssh-server_removed" />
       </criteria>
 {{%- endmacro %}}
+
+{{#
+  High level macro which checks configuration in an INI file.
+  This macro can take eight parameters:
+    - path (String): Path to the configuration file to be checked.
+    - section (String): The parameter will be checked only within the given section defined by [section].
+    - parameter (String): The parameter to be checked in the configuration file.
+    - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
+    - missing_parameter_pass (boolean): If set, the check will also pass if the parameter is not present in the configuration file (default is applied).
+    - application (String): The application which the configuration file is being checked. Can be any value and does not affect the OVAL check.
+    - multi_value (boolean): If set, it means that the parameter can accept multiple values and the expected value must be present in the current list of values.
+    - missing_config_file_fail (boolean): If set, the check will fail if the configuration is not existent in the system.
+#}}
+{{%- macro oval_check_ini_file(path='', section='', parameter='', value='', missing_parameter_pass=false, application='', multi_value=false, missing_config_file_fail=false) %}}
```

Comment:
As devel doc was added for these macros, please add doc for this macro as well. Here: https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer_guide.adoc#macros

Reply:
I've seen that you updated accordingly. Thanks.